src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -415,16 +415,30 @@ <h3 *ngIf="displaySubsection(showQuestion, 7, 10)">Adding Comments to Responses<
               View the results of a session
             </li>
             <li>
-              Click <button class="btn btn-primary btn-s">Edit View</button> and change the view type to <b>Group by - Giver &gt; Recipient &gt; Question</b> or <b>Group by - Recipient &gt; Giver &gt; Question</b>
+              Click <button class="btn btn-light btn-s dropdown-toggle">View</button> dropdown and change the view type to <b>Group by - Giver &gt; Recipient &gt; Question</b> or <b>Group by - Recipient &gt; Giver &gt; Question</b>
             </li>
             <li>
-              Click the <i class="fas fa-comment-alt"></i> icon on the right-hand side of the response you would like to comment on
+              Click the <button type="button" class="btn btn-secondary btn-sm"><i class="fas fa-comment"></i>&nbsp;<i class="fas fa-plus"></i></button> icon on the right-hand side of the response you would like to comment on
             </li>
             <li>
               Fill in the form, which will appear similar to the example below
             </li>
           </ol>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <div class="card-body">
+              <tm-comment-edit-form
+                [mode]="CommentRowMode.ADD"
+                [model]="exampleModel"
```

Comment:
```suggestion
                [(model)]="exampleModel"
```

Reply:
This is a nice syntax sugar called BANANA syntax.

https://medium.com/@ofirrifo/custom-angular-component-using-two-way-data-binding-aka-banana-in-a-box-syntax-9eb06b8cfb09